src/OVAL/probes/unix/linux/rpminfo.c

```diff
@@ -143,7 +143,7 @@ static void pkgh2rep (Header h, struct rpminfo_rep *r)
 
 	if (regexec(&g_keyid_regex, str, 1, keyid_match, 0) != 0) {
 		sid = NULL;
-		dW("Failed to extract the Key ID value: regex=\"%s\", string=\"%s\"",
+		dI("Failed to extract the Key ID value: regex=\"%s\", string=\"%s\"",
```

Comment:
This looks like it should be a developer level warning, it has regular expression and a string.
It is not clear to me what is criteria.